docs/pages/rigid-object-tutorial.rst

```diff
@@ -0,0 +1,154 @@
+In this example, a sphere object template is loaded and the object is instanced in the scene above the table.
+When the simulation is stepped, it falls under the force of gravity and reacts to collisions with the scene.
+
+.. include:: ../../examples/tutorials/rigid_object_tutorial.py
+    :code: py
+    :start-after: # [basics]
+    :end-before: # [/basics]
+
+.. image:: images/rigid-object-tutorial-images/sim_basics.gif
+    :width: 20em
+
+Forces and torques can be applied to the object with :ref:`Simulator.apply_force` and :ref:`Simulator.apply_torque`.
+Instantanious initial velocities can also be set with :ref:`Simulator.set_linear_velocity` and :ref:`Simulator.set_angular_velocity`.
+
+In the example below, a constant anti-gravity force is applied to the boxes' centers of mass (COM) causing them to float in the air.
+A constant torque is also applied which gradually increases the angular velocity of the boxes.
+A sphere is then thrown at the boxes by applying an initial velocity.
+
+Note that forces and torques are treated as constant within each call to :ref:`Simulator.step_physics` and are cleared afterward.
+
+.. include:: ../../examples/tutorials/rigid_object_tutorial.py
+    :code: py
+    :start-after: # [dynamic_control]
+    :end-before: # [/dynamic_control]
+
+.. image:: images/rigid-object-tutorial-images/dynamic_control.gif
+    :width: 20em
+
+
+`Kinematic Object Placement`_
+=============================
+
+Often it is enough to set the desired state of an object directly.
+In these cases the computational overhead of running full dynamic simulation may not be necessary to achieve a desired result.
+Setting the object to :ref:`habitat_sim.physics.MotionType.KINEMATIC` with :ref:`Simulator.set_object_motion_type` specifies that the object's state will be directly controlled.
+
+In the example below, a kinematic can is placed in the scene which will not react to physical events such as collision with dynamically simulated objects.
+However, it will still act as a collision object for other scene objects as in the following example.
+
+.. include:: ../../examples/tutorials/rigid_object_tutorial.py
+    :code: py
+    :start-after: # [kinematic_interactions]
+    :end-before: # [/kinematic_interactions]
+
+.. image:: images/rigid-object-tutorial-images/kinematic_interactions.gif
+    :width: 20em
+
+
+`Kinematic Velocity Control`_
+=============================
+
+
+To move a kinematic object, the state can be set directly before each simulation step.
+This is useful for synchronizing the simulation state of objects to a known state such as a dataset trajectory, input device, or motion capture.
+
+.. include:: ../../examples/tutorials/rigid_object_tutorial.py
+    :code: py
+    :start-after: # [kinematic_update]
+    :end-before: # [/kinematic_update]
+
+.. image:: images/rigid-object-tutorial-images/kinematic_update.gif
```

Comment:
1. These videos work at my end. 

2. I really like these examples, but can we show the LoCoBot mesh moving on the floor? Like these https://github.com/facebookresearch/habitat-sim/pull/486

I suspect a lot of people will come looking to see how to control robots. This might also be a good place to talk about attaching a scene node to an instanced physics object so people can record videos from attached/overhead camera as shown here: #492.

Reply:
Thanks for this suggestion, I've added a new section to the tutorial demonstrating this.